src/services/login-handler.js

```diff
@@ -36,6 +36,7 @@ function LoginHandler({
 
     if (TWO_FACTOR_SECRET_SALT === undefined) {
       logger.error('Cannot use the two factor authentication because the environment variable "FOREST_2FA_SECRET_SALT" is not set.');
+      logger.error('You can generate it using this command: `$ openssl rand -hex 10`');
```

Comment:
IMO this info should be in the doc or in the feature onboarding steps not in an error log.
That's why I removed it in the first implementation.
Once a doc section has been created, we could had the link in this error message.

If you really want to add this, I would concatenate the new sentence to the first error log message to make it obvious that they are related.
(In the case a third error message appears for example)